nav2_smac_planner/src/smac_planner_hybrid.cpp

```diff
@@ -338,6 +338,10 @@ nav_msgs::msg::Path SmacPlannerHybrid::createPlan(
   }
 
   // Set collision checker and costmap information
+  _collision_checker.setFootprint(
```

Comment:
Can you set to update only when changed? Recomputing each time with footprint precomputing will add unnecessary latency.

Reply:
The footprint change check is already done in this function:
https://github.com/ros-planning/navigation2/blob/914d881f6e154d8d7bb1975892d4642eaf4c7366/nav2_smac_planner/src/collision_checker.cpp#L51-L67